app/controllers/middleware_server_controller.rb

```diff
@@ -45,6 +45,12 @@ class MiddlewareServerController < ApplicationController
                                     :hawk  => N_('Not deploying to Hawkular server'),
                                     :msg   => N_('Deployment initiated for selected server(s)'),
                                     :param => :file
+    },
+    :middleware_add_jdbcdriver  => {:op    => :add_middleware_jdbc_driver,
+                                    :skip  => false,
+                                    :hawk  => N_('Not installing new JDBC Driver to Hawkular server'),
```

Comment:
The `skip`is to decide wether to run or not (skip) the operation for Hawkular type servers. So you can't Suspend/Stop the provider itself. It's used [here](https://github.com/ammendonca/manageiq/blob/ce960011803ac4d7795732cbb20425b30c6b7484/app/controllers/middleware_server_controller.rb#L158-L159).

Indeed for this operation (Add JDBC Driver) it doesn't seem to make much sense as it may always run on the provider, thus the skip being false. Also the message is not proper, as concatenating with the rest will make it weird. Will refactor.